userDetails.py

```
@@ -0,0 +1,17 @@
+full_name = input("Enter your full name: ")
+country = input("Enter your country: ")
+language = input("Enter your preferred language: ")
+
+# Create and print welcome message
+message = f"""
+Welcome, {full_name}!
+
+Your username is {username}, and it's great to have someone from {country} here.
+We see that you prefer communicating in {language}, which is awesome.
+
+We’re glad to have you onboard—enjoy your experience!
```

Ruff (0.15.6)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)